rl4co/envs/routing/mtvrp/env.py

```diff
@@ -450,7 +455,7 @@ def _make_spec(self, td_params: TensorDict):
     def check_variants(td):
         """Check if the problem has the variants"""
         has_open = td["open_route"].squeeze(-1)
-        has_tw = (td["time_windows"][:, :, 1] != float("inf")).any(-1)
+        has_tw = (td["time_windows"][:, :, 1] != 4.6).any(-1)
```

Comment:
Same as the discussion with the `_default_time_window()`, Changing this bound in the setting will need to modify this part. Any reason for this hardcode?

Reply:
Avoid numerical issues during training as it will go through embedding, but I will have a check, the inf would be more general.